docs/docs-ref-autogen/excel/excel.shape.yml

```diff
@@ -168,7 +168,9 @@ items:
 
           ```typescript
           await Excel.run(async (context) => {
-              const sheet = context.workbook.worksheets.getItem("Shapes");
+              context.workbook.worksheets.getItemOrNullObject("Shapes").delete();
```

Comment:
Just a sanity check. I understand that `getItemOrNullObject` won't throw an exception if there is no "Shapes" sheet. Have you tested to verify that worksheet.delete() won't throw if the sheet is null?

Reply:
It does not, thankfully. You can test this in the SL samples right now.